spec/values/calculation/rem.hrx

```diff
@@ -0,0 +1,324 @@
+<===> README.md
+Most of the same behavior tested for `calc()` applies to `rem()`, but for
+terseness' sake isn't tested explicitly.
+
+<===>
+================================================================================
+<===> units/none/input.scss
+a {b: rem(7, 3)}
+
+<===> units/none/output.css
+a {
+  b: 1;
+}
+
+<===>
+================================================================================
+<===> units/compatible/input.scss
+a {b: rem(5px, 3px)}
+
+<===> units/compatible/output.css
+a {
+  b: 2px;
+}
+
+<===>
+================================================================================
+<===> units/real_and_unknown/input.scss
+a {b: rem(5px, 3%)}
+
+<===> units/real_and_unknown/output.css
+a {
+  b: rem(5px, 3%);
+}
+
+<===>
+================================================================================
+<===> units/unknown/input.scss
+a {
+  b: rem(1%, 2%);
+}
+
+<===> units/unknown/output.css
+a {
+  b: 1%;
+}
+
+<===>
+================================================================================
+<===> units/fake/input.scss
+a {
+  b: rem(1foo, 2bar);
+}
+
+<===> units/fake/output.css
+a {
+  b: rem(1foo, 2bar);
+}
+
+<===>
+================================================================================
+<===> units/same_fake/input.scss
+a {
+  b: rem(1foo, 2foo);
+}
+
+<===> units/same_fake/output.css
+a {
+  b: 1foo;
+}
+
+<===>
+================================================================================
+<===> units/real_and_fake/input.scss
+a {
+  b: rem(1px, 2bar);
+}
+
+<===> units/real_and_fake/output.css
+a {
+  b: rem(1px, 2bar);
+}
+
+<===>
+================================================================================
+<===> positive_and_negative/input.scss
+a {b: rem(2, -5)}
+
+<===> positive_and_negative/output.css
+a {
+  b: -5;
```

Comment:
This output looks wrong. From [the CSS spec](https://drafts.csswg.org/css-values-4/#funcdef-mod), `mod()` and `rem()` "return the difference between A and the nearest integer multiple of B either above or below A." That means `rem(2, -5)` should be either -3 (the difference between 2 and the nearest integer multiple of -5 above 2, which is 5) or 2 (the difference between 2 and the nearest integer multiple of -5 _below_ 2, which is 0). It definitely shouldn't be -5.

Specifically, since `rem()` "chooses the integer multiple of B that puts the value between zero and -B, avoiding changing the sign of the value" when the two arguments have different signs, this should be 2 and the `negative_and_positive` case should be -3.

Reply:
I think this may be due to a bug in [the Sass spec](https://github.com/sass/sass/blob/main/accepted/calc-functions.md?plain=1#L376), which says that if the signs are different, then `rem(dividend, modulus)` should equal `mod(dividend, modulus) - dividend`, when it should equal `mod(dividend, modulus) - modulus`